__tests__/test.js

```diff
@@ -225,3 +225,21 @@ describe("DELETE /api/comments/:comment_id", () => {
       });
   });
 });
+
+describe("GET /api/users", () => {
+  test("return all users", () => {
+    return request(app)
+      .get("/api/users")
+      .then((users) => {
+        expect(users.status).toBe(200);
```

Comment:
This parameter has been named `users` but it is the `response` object which could cause confusion, the _users_ data should be on a key of `users` inside the `response` object